### PR TITLE
Add paper wallet to example

### PR DESCRIPTION
**Pull request by NicolasDP**
@vincenthz , just pushed the [Paper Wallet](https://github.com/input-output-hk/cardano-specs/blob/master/proposals/0001-PaperWallet.md) support to the example too. You might want to check it out.

screenshot from 2018-03-08 18-05-26

### CODE REVIEW

**Comment by DominikGuzei** [comment]
@NicolasDP please let's discuss on slack

File: js-example/src/Components/Bindings/HdWallet.js [hunk 1]
```diff
@@ -44,11 +47,24 @@ function base16(u8) {
     return b16;
 }
 
+exports.scrambleImpl = function (entropy) {
+    return function (pass) {
+        var pass_buffer = utf8Encoder.encode(pass);
+        var iv = randomBytes(4);
+        try {
+            return window.Module.PaperWallet.scramble (iv, pass_buffer, entropy);
```
Comment (DominikGuzei) [inline js-example/src/Components/Bindings/HdWallet.js]: Please change this to `window.CardanoCrypto.PaperWallet.scramble`

Comment (darko-mijic) [reply js-example/src/Components/Bindings/HdWallet.js]: Yes, please do.

**Comment by darko-mijic** [comment]
@vincenthz Let's not merge PRs without peer reviews.

**Comment by vincenthz** [comment]
@darko-mijic js-example is for us to test the crypto APIs. I saw it in action and reviewed it, so pretty happy to merge this.

**Pull request merged**